CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ All notable changes to this project are documented in this file.
  - Added `From<&str>` and `From<SharedString>` to `StandardListViewItem` to make creation and modification of `StandardListView`'s models easier.
  - Added `on_close_requested` function to `Window` to register callbacks that are emitted when the user tries to close a window.
  - Added `VecModel::set_vec` to replace the entire contents with new data.
+ - Added a `layer` boolean property that can be applied to any element, to hint to the renderer that it should cache the element and its children
```

Comment:
we forgot the changelog entry for the opacity layer fix

Reply:
I've pushed that into master now. Thanks for noticing :)